### DISCUSSION TITLE
Intermittent angry unicorn

### DISCUSSION THREAD

**Discussion by QuackingPlums** [community/community#23444] [Other Feature Feedback, Questions, & Ideas]
<p>Hi. I’ve had one of my gh-pages hosting a simple info/status page for a long time (several years), but a recent downstream configuration change required me to edit the URL of a media asset that was embedded in the HTML.</p>
<p>I’ve had an intermittent angry unicorn since committing that change (I’ve tried via both terminal/command line push and editing the HTML file in the github UI).</p>
<p>I say ‘intermittent’ because it does VERY occasionally work, but I would guess that for about 90% of the time I get the unicorn.</p>
<p>Has there been a change to how gh-pages work that means I have to do something new? I’ve searched the forum for other instances of this symptom but all I’ve found are reports that it got escalated, followed by “it seems fixed now”. I’ve been subject to the angry unicorn for over a week now.</p>
<p>This is using a custom subdomain, if that’s relevant (i.e.: <a href="http://mypage.mydomain.com" rel="noopener nofollow ugc">mypage.mydomain.com</a>). There’s nothing noteworthy in my page that I can imagine would break this functionality (although of course I would say that — if I could see it then I’d fix it :grinning: )</p>
<p>Any suggestions appreciated.</p>

**Comment by jfullstackdev** [comment]
<p>seems to me it is not on GitHub side</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/1/416d1ad7dbff91d8be39cb9bad79ab6bfc2b3cf7.png" data-download-href="/uploads/short-url/9kMN57HHIbuNMjpRYmadQPWN9Yj.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077117-b2fa1690-fea3-4ba2-87f9-e9c0e490f07a.png" alt="image" data-base62-sha1="9kMN57HHIbuNMjpRYmadQPWN9Yj" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/1/416d1ad7dbff91d8be39cb9bad79ab6bfc2b3cf7_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/1/416d1ad7dbff91d8be39cb9bad79ab6bfc2b3cf7_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/1/416d1ad7dbff91d8be39cb9bad79ab6bfc2b3cf7_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/1/416d1ad7dbff91d8be39cb9bad79ab6bfc2b3cf7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 132 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>if you take a look at Developer Tools, you’ll see</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181077118-05c9baf5-f75e-40fa-92e6-1d4373332987.png" alt="image" data-base62-sha1="mmrdsDz8hhhiNnjdigOujYnTYVe" width="496" height="268"></p>
<p>the server responded 503, it’s service unavailable, that is not GitHub, that is the domain you are using</p>

**Comment by yoannchaudet** [comment]
<p>Hello <a class="mention" href="/quackingplums">@QuackingPlums</a> and welcome here!</p>
<p>This one has left me scratching my head for a bit. I think your site was so old, it ended in a very odd state. I spent a bit of time cleaning things up and you should be all good to go now. Sorry for the few manual rebuilds I had to trigger and for the annoyance.</p>
<p>Let me know if that helps,</p>

**Comment by queenofcorgis** [comment]
<p>Thanks <a class="mention" href="/yoannchaudet">@yoannchaudet</a> for jumping in and for cleaning :broom: <a class="mention" href="/quackingplums">@QuackingPlums</a>  ahoy and welcome - please do let us know how things are going :smile:</p>

**Comment by QuackingPlums** [comment]
<p>Many thanks for this <a class="mention" href="/yoannchaudet">@yoannchaudet</a> ! Yes, the site is very old and I haven’t had to touch it in a while — I’m embarrassed to confess that I’m no longer considered an engineer by vocation so git/github is something that I don’t get to use very much any more!</p>
<p>I did see the 503 error in the logs but the favico file hadn’t changed in 8 years so I dismissed it as just another symptom of the site only serving intermittently.</p>
<p>Thanks again for your help and to everybody else for jumping in to take a look! :heart:</p>

**Comment by michaelcmartin** [comment]
<p>I’ve been having a very similar issue with <a href="http://michaelcmartin.github.io/megaforge/" rel="noopener nofollow ugc">http://michaelcmartin.github.io/megaforge/</a> complete with 503 error, but I do <em>not</em> have a custom domain name here.</p>
<p>My other site (<a href="http://michaelcmartin.github.io/Ophis/" class="inline-onebox" rel="noopener nofollow ugc">The Ophis Assembler</a>) which I’ve also generally never updated seems to be OK.</p>

**Comment by jfullstackdev** [comment]
<p>maybe this one is related too</p>
<aside class="quote" data-post="1" data-topic="253497">
  <div class="title">
    <div class="quote-controls"></div>
    
    <a href="https://github.community/t/github-pages-error-the-unicorns-have-taken-over/253497">GitHub Pages Error: The Unicorns have taken over.</a> <a class="badge-wrapper  bullet" href="/c/github-help/github-pages/31"><span class="badge-category-parent-bg" style="background-color: #6a737d;"></span><span class="badge-category-bg" style="background-color: #c8e1ff;"></span><span style="" data-drop-close="true" class="badge-category clear-badge" title="Questions and discussion about creating a website directly from a repository on GitHub.com.">GitHub Pages</span></a>
  </div>
  <blockquote>
    I created my <a href="http://agryman.github.io" rel="noopener nofollow ugc">agryman.github.io</a> repo in 2015 but didn’t use it much. I now have some content I need to publish, but for the last few days I get the unicorn error. What do I need to do to enable my website? Thanks. 
We’re having a really bad day. 
The Unicorns have taken over. We’re doing our best to get them under control and get GitHub back up and running.
  </blockquote>
</aside>

**Comment by knowshan** [comment]
<p>Thank you <a class="mention" href="/quackingplums">@QuackingPlums</a> and <a class="mention" href="/yoannchaudet">@yoannchaudet</a> on posting question and answers here in the community. Looks like I’m not alone running into this issue.</p>
<p>My site with last update (&gt; 10 years ago) was running fine. After I published a test post this week, Github unicorn has got angry. When I published same site to netlify (still with old css and JS), it worked fine.</p>
<p>I’m planning to update my site anyway, but it would be good to know how we can essentially ‘reset’ site.</p>

**Comment by michaelcmartin** [comment]
<p>This came up on the link above; you can ‘reset’ the site by disabling GitHub Pages and then re-enabling it. If you are using a <code>gh-pages</code> branch for this, you’ll have to rename the branch (<code>git checkout -b website</code> will do this fine) and then remove the gh-pages branch before you can actually disable the legacy pages.</p>
<p>I’ve verified this workaround on my Megaforge site.</p>
